java/ql/src/experimental/Security/CWE/CWE-749/UnsafeAndroidAccess.qhelp

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+  <overview>
+    <p>Android WebViews that allow loading urls controlled by external inputs and whose JavaScript interface is enabled are potentially vulnerable to cross-site scripting and sensitive resource disclosure attacks.</p>
+    <p>WebSettings of WebViews with setAllowFileAccessFromFileURLs or setAllowUniversalAccessFromFileURLs enabled must not load any untrusted web content.</p>
+    <p>Enabling this setting allows malicious scripts loaded in a file:// context to launch cross-site scripting attacks, either accessing arbitrary local files including WebView cookies, session tokens, private app data or even credentials used on arbitrary web sites.</p>
+    <p>This query detects the following two scenarios:</p>
+    <ol>
+      <li>Vulnerability introduced by WebViews with JavaScript enabled and remote inputs allowed.</li>
+      <li>High precision vulnerability when allowing universal resource access is also enabled. The setting was just deprecated in API level 30 (Android 11) thus most devices are still affected given that Android phones don't get timely version updates like iPhones.</li>
+    </ol>
+  </overview>
+
+  <recommendation>
+    <p>Only allow trusted web contents to be displayed in WebViews when JavaScript is enabled. And disallow universal resource access in WebSetting to reduce the attack surface .</p>
```

Comment:
```suggestion
    <p>Only allow trusted web contents to be displayed in WebViews when JavaScript is enabled. And disallow universal resource access in WebSettings to reduce the attack surface.</p>
```

Reply:
Thanks @Marcono1234 . The proposed change makes the wording consistent with the class name thus helps to clarify. I've made the change.

Reply:
No problem. Also there is a redundant space after "surface"; might not be that easy to spot in the diff of my previous comment.

Reply:
Thanks @Marcono1234 . I won't make a separate commit for the extra space, but will make the change together with other changes to be made.

Reply:
Are you planning to do more changes or do you mean in case there are more changes requested by other review comments?
If you mean the latter, you could also amend your last commit and then force push (`--force-with-lease`).

Reply:
Thanks @Marcono1234 for the advice. I tried but didn't get it right so I just committed a regular commit to fix the issue.